src/main/java/uk/gov/justice/hmpps/prison/repository/BookingRepository.java

```diff
@@ -605,6 +606,19 @@ public ScheduledEvent getBookingAppointment(final Long bookingId, final Long eve
                 EVENT_ROW_MAPPER);
     }
 
+    public Optional<ScheduledEvent> getBookingAppointmentByEventId(final long eventId) {
+        final var results = jdbcTemplate.queryForList(BookingRepositorySql.GET_BOOKING_APPOINTMENT_BY_EVENT_ID.getSql(), createParams("eventId", eventId));
```

Comment:
this doesn't look right, this isn't used?

Reply:
You're right.  That was meant to be temporary.

Reply:
I think it's used here, but could very well be wrong https://github.com/ministryofjustice/prison-api/pull/763/files#diff-c9ddc50428a878082834636765a70c1779f2d94db7885e1a59e940d7b007171dR126